packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNavIcon/__snapshots__/TeachingPopoverCarouselNavIcon.test.tsx.snap

```diff
@@ -0,0 +1,14 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TeachingPopoverCarouselNavIcon renders a default state 1`] = `
+<div>
+  <div
+    class="fui-TeachingPopoverCarouselNavIcon"
+  >
+    <button
+      class="fui-TeachingPopoverCarouselNavIcon__navButton"
+      type="button"
+    />
+  </div>
```

Comment:
The underlying button relies on opacity for styling, but we want no opacity on the border/focus styles.

Usually, I would just achieve this via background opacity value, but since we use tokens it would be a bit weird string logic to ensure, so just used css opacity on internal button with wrapper as focus/border.